packages/vm/src/runTx.ts

```diff
@@ -31,7 +31,7 @@ const debugGas = createDebugLogger('vm:tx:gas')
  */
 export async function runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
   // tx is required
-  if (isFalsy(opts.tx)) {
+  if (opts.tx === undefined) {
```

Comment:
We should most likely enforce this in the `opts` type..?

Reply:
Indeed, removed.